be-traveltrove/src/main/java/com/traveltrove/betraveltrove/dataaccess/tour/Tour.java

```diff
@@ -24,17 +24,7 @@ public class Tour {
 
     private String tourId;
     private String name;
-    private LocalDate startDate;
-    private LocalDate endDate;
-    private String overallDescription;
-    private boolean available;
-    private double price;
-    private int spotsAvailable;
+    private String description;
 
-    private List<String> cities;
-    private List<String> reviews;
-    private List<String> bookings;
-
-    private String image;
-    private String itineraryPicture;
+    private List<TourEvents> tourEvents;
```

Comment:
Here, the relationship between tour events should indeed be a list, because of the nature of the many to many relationship between tours and events. Well done!